redbot/cogs/trivia/data/lists/prince.yaml

```diff
@@ -0,0 +1,221 @@
+- Money Don't Matter 2 Night
+- Money Don't Matter 2Night
+(Identify the Prince song) Drop your bombs on each other, in the act of saving face:
+- Resolution
+(Identify the Prince song) The main problem with people is they never do what they say:
+- Resolution
+(Identify the Prince song) Make a promise this very night that they will no longer pose a threat to either one of us:
+- All The Midnights In The World
+(Identify the Prince song) Together with love for one another, we shall make them known:
+- All The Midnights In The World
+(Identify the Prince song) I can't disguise the pounding of my heart, it beats so strong:
+- Take Me With U
+(Identify the Prince song) You're sheer perfection:
+- Take Me With U
+(Identify the Prince song) I know your heart is beating, my drummer tells me so:
+- Free
+(Identify the Prince song) Be glad that you are free, free to change your mind:
+- Free
+(Identify the Prince song) She wears a long fur coat of mink, even in the summer time:
+- The Glamorous Life
+(Identify the Prince song) She's got big thoughts, big dreams and a big brown Mercedes Sedan:
+- The Glamorous Life
+(Identify the Prince song) Have to catch an early train, got to be at work by nine:
+- Manic Monday
+(Identify the Prince song) Blame it on the train, but the boss is already there:
+- Manic Monday
+(Identify the Prince song) He left her with a baby and another one on the way:
+- I Could Never Take The Place Of Your Man
+(Identify the Prince song) She asked me if we could be friends and I said ooh, honey, baby, that's a dead end:
+- I Could Never Take The Place Of Your Man
+(Identify the Prince song) Big cars and women and fancy clothes will save your face, but it won't save your soul:
+- My Name Is Prince
+(Identify the Prince song) Would our Lord be happy if he came today?:
+- My Name Is Prince
+(Identify the Prince song) The sky was all purple, there were people running everywhere trying to run from the destruction:
+- 1999
+(Identify the Prince song) Everybody's got a bomb, we could all die anyday:
+- 1999
+(Identify the Prince song) There is a woman who sits all alone by the pier, her husband was naughty and caused his wife so many tears:
+- Paisley Park
+(Identify the Prince song) See the man cry as the city condemns where he lives. Memories die, but taxes, he'll still have to give.:
+- Paisley Park
+(Identify the Prince song) They could take a walk down the ocean side, make a wish on every wave:
+- The Morning Papers
+(Identify the Prince song) They could find a carousel and ride, or kiss in every cave:
+- The Morning Papers
+(Identify the Prince song) Don't my kisses please you right?:
+- The Beautiful Ones
+(Identify the Prince song) You were so hard to find, the beautiful ones, they hurt you every time:
+- The Beautiful Ones
+(Identify the Prince song) She took me to her castle and I just couldn't believe my eyes:
```

Comment:
This song *might* not suitable for core red due to the content. Needs discussion.

Reply:
Removed :+1: